kdwsdl2cpp/src/main.cpp

```diff
@@ -123,6 +147,40 @@ int main(int argc, char **argv)
                 return 1;
             }
             nameSpace = argv[arg];
+        } else if (opt == QLatin1String("-namespaceMapping")) {
+            ++arg;
+            if (!argv[arg]) {
+                showHelp(argv[0]);
+                return 1;
+            }
+            QString mapping = argv[arg];
+            if (mapping.startsWith('@')) {
+                QString mappingFileName = mapping.mid(1);
+                QFile file(mappingFileName);
+                if (!file.open(QIODevice::ReadOnly)) {
+                    fprintf(stderr, "Error reading %s: %s\n", mappingFileName.toLatin1().data(), file.errorString().toLatin1().data());
```

Comment:
`QFile::encodeName(mappingFileName).constData()` would be more correct (in case of non-latin1 chars in the path)